### ISSUE TITLE
Huge loss result

### ISSUE THREAD

**Issue by Skye-Jiang** [pz-white/COM6911-DTI-2021#12]
Hi @pz-white 
We meet a few problems and do you mind having a look? We tried to make predictions in Pykale based on clustering results, but we got high loss values, as shown in the picture. We have tried many things, but we couldn't solve the problem.  For example:
1. no matter if we remove duplicate "Drug" and 'Target' rows
2. no matter we use PCA or not
3. the data splitting method also tried random split cluster or after balanced the number of sample in every cluster.
4. and we also tried setting the y_log as True or False
However, the three cluster method loss results are all extremely high. 

Could you please take a look at our code if we made mistake?

**Comment by pz-white** [comment]
Hi @Skye-Jiang, thanks and I have several questions:
- How many epoches did you set in you experiment? I see the default is 1 in your code (config.py) but it should be 100 in a complete running.
- Could you save the new bindingDB data after clustering and compare it with original one? I see you should only add **Cluster** column as the clustering result in the original TDC dataframe, you may check whether the new data is consistent to original one before cluserting except added **Cluster** column.
Thanks.

**Comment by liudaotan** [comment]
Hi @pz-white , thanks for your suggestion. I have tried 100 epochs. The loss value after 100 epoch  is reduced to about 1/10 of the first epoch loss. However, the final loss value is still very large.
My team menber have tried to pass the same data to pykale as it's required as well and she have dropped all the extra columns which we have added earlier for clustering computation. But unfortunately, the loss values are still very large.

![pasted image 0](https://user-images.githubusercontent.com/72572210/118034278-b718fd80-b361-11eb-8d53-3c4a3bb49841.png)

**Comment by Skye-Jiang** [comment]
This is the train dataframe after data splitting. 

![dataframe](https://user-images.githubusercontent.com/69766451/118040152-20503f00-b369-11eb-985f-3841fcd36fd5.png)

**Comment by pz-white** [comment]
Hi @liudaotan @Skye-Jiang 
- Coud you check the logger with tensorboard? I cannot see the training process from your screenshot but the epoch 99 shows that the traning loss is very small and it is convergent.
- If the training loss is normal, the model traning is correct and the issue may be caused by the data itself. Could you check the original data and processed one after clustering? Not only the train dataframe, I think you need to check the whole dataset.

**Comment by shrutidudwadkar** [comment]
Hi @pz-white 

Thanks a lot for your input regarding the y_log. I have now fixed the issue and the loss is as follows for the first epoch.
<img width="720" alt="Screenshot 2021-05-13 at 16 05 32" src="https://user-images.githubusercontent.com/51436405/118146233-4bd53700-b406-11eb-87d2-b8f0d3ce173f.png">

I guess we can close this issue now.
